packages/backend/src/workers/action.ts

```diff
@@ -124,6 +124,22 @@ worker.on('failed', async (job, err) => {
       job: job.data,
     },
   )
+
+  if (
+    !(err instanceof UnrecoverableError) &&
+    job.attemptsMade < MAXIMUM_JOB_ATTEMPTS
```

Comment:
can we consolidate this condition and the condition in line 148-149 together. That and / or refactor to `else`. Unconsolidated correlated conditions feel like a footgun.

e.g.
```
const isRetriable = !(err instanceof UnrecoverableError) && job.attemptsMade < MAXIMUM_JOB_ATTEMPTS
if (isRetriable) {
  // Add span tag
  const span = ...
} else {
  // send error email, etc.
  const isEmailSent = ...
}
```

Reply:
just checking but the else shouldn't be there in this example given right?

Reply:
I was thinking of refactoring the entire function in my example, so the `else` should be there.